otwcode/otwarchive
# AO3-4159 Add last login information to admin user display

## Issue

https://otwarchive.atlassian.net/browse/AO3-4159

## Purpose

Adds the information devise has added ( last login and ip address ) to the admin user abuse page.

This replace https://github.com/otwcode/otwarchive/pull/1975

## Testing Instructions

Have a user login. 

Login to an admin account and check ip address, time and date of user last login is correct.
